### ISSUE TITLE
Can't share both text and image iOS

### ISSUE THREAD

**Issue by matthewfx** [zhouteng0217/ShareExtend#30]
`ShareExtend.share(path, "test");
`
It just shares an image.

**Comment by zhouteng0217** [comment]
@matthewfx  Yeah, it is not support for sharing both text and image.  It just providers a way for sharing with system ui  and limited by system api.

**Comment by matthewfx** [comment]
Hopefully, you will add it one day - thanks!

**Comment by JEuler** [comment]
@matthewfx you can have a little trick here (I'm using it in my app) - just copy the text content to the clipboard and show user a toast, like "you can insert caption from clipboard" etc. 

`Clipboard.setData(ClipboardData(text: yourText));`

**Comment by rignaneseleo** [comment]
Any news about this?

**Comment by DeepashreeB** [comment]
Cant share video with Text also
